### PR TITLE
Changed color of visited links

### LINKED ISSUE #3415: Ahem, visited link color/assignment to certain buttons..
Screenshot 2022-06-14 at 11 49 17

### PR DESCRIPTION
**Pull request by northwestwitch**
Tentative fix #3415 

How do they look like this?

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN, CR

### CODE REVIEW

**Comment by dnil** [comment]
Artsy! It works, but eh, perhaps not entirely synced up. Another solution that flew through my head was adding a class to the variantS list page variant links (you know, the rank or variant description ones depending), and have that class be the only one that gets the visited change. Honestly it just looks a bit weird on the buttons and drop-down menu with some links visited until you have visited them all...

Screenshot 2022-06-14 at 13 57 30


Screenshot 2022-06-14 at 13 57 11

**Review by dnil — APPROVED** [review]
This solves the issue, but the style nitpicking department may have a pause over it.

**Comment by northwestwitch** [comment]
> Honestly it just looks a bit weird on the buttons and drop-down menu with some links visited until you have visited them all...

Agreed, but then why don't we just fix the buttons to not see like links instead?

**Comment by dnil** [comment]
Yes, but there are maaaany buttons..

**Comment by northwestwitch** [comment]
Also the ACMG buttons have the problem

File: scout/server/static/bs_styles.css [hunk 1]
```diff
@@ -33,7 +33,11 @@ table.dataTable {margin-bottom: 0}
 }
 
 a:visited {
-	color: var(--bs-purple);
+	color: var(--bs-cyan);
+}
+
+a.btn:visited {
+  color: white;
```
[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/static/bs_styles.css:40]: Right.. We could do only btn-secondary, and perhaps also another solution for the dropdown links?

[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/static/bs_styles.css:40]: Aaaah these ones: 

image

UGLY!!!

[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/static/bs_styles.css:40]: Yeah, afaik that was what we had prior to bs5, only with a more custom visited color (and only in light mode).

[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/static/bs_styles.css:40]: Fixed in the laziest way:

image

Feel free to push changes to this branch if you want to do it in another way

**Comment by dnil** [comment]
👍 I'll check out and click around a little!

File: scout/server/static/bs_styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/static/bs_styles.css:40]: Ok but dark mode is not that visible any more:

image

But I think that that dropdown might be fixed to have the same color as the top of the window, where it comes from, no?

File: scout/server/static/bs_styles.css [hunk 2]
```diff
@@ -33,7 +33,15 @@ table.dataTable {margin-bottom: 0}
 }
 
 a:visited {
-	color: var(--bs-purple);
+	color: var(--bs-cyan);
```
Comment (dnil) [inline scout/server/static/bs_styles.css:36]: Actually you could revert to purple now after fixing the button links..

Comment (northwestwitch) [reply scout/server/static/bs_styles.css:36]: Yes, but perhaps cyan is more visible than purple?

Purple:

image


Cyan:

image

Maybe not as much on white tho:

image

**Pull request merged**